src/main/java/com/google/devtools/build/lib/remote/blobstore/GrpcBlobStore.java

```diff
@@ -11,17 +11,13 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+package com.google.devtools.build.lib.remote.blobstore;
```

Comment:
how do you feel about creating a package ...remote.blobstore.grpc? We have one already for http, so it feels natural to add the second one.

Reply:
Yes, we can totally try it. It will only be one file so far though.